### PR TITLE
Medical Intern Alt-Titles

### PR DESCRIPTION
**Pull request by KingOfThePing**
**WHAT?**
To make the learning role of medical intern more logical, this PR adds two new alt titles:

- Surgical Resident
- Medic Trainee

For surgeons and first responders respectively. It is tiring to ask interns everyone (or tell as intern every round) what your field of study is. In accordance to this medical interns now also have access to the first responder's quarters.

The existing title or Medical Intern is reserved for more general physician residents or students in an early stage without specialization yet.

**WHY?**
Alt-titles like these do not differentiate the job in it's responsibilities and thus do not create discrepancies in expectations. It is merely an RP tool for character development, put into a logical perspective. IRL students who are in their residency are also titles as the respective residency - e.g. "Surgical Resident", so everyone knows in what department they are used as.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
Eh, don't think this is really needed. The last time we had an alt-title with no real different responsiblities was *nurse*, and it got removed for being too useless. Think the same will happen here.

**Comment by KingOfThePing** [comment]
The problem with nurse was that it was an alt-title for doctors IIRC and it just overlapped with general duties, like 100%. Here it is specifically to differentiate the interns in their field of work

**Comment by desvenlafaxine** [comment]
I support this, but I'm not 100% sold on the names.

**Comment by KingOfThePing** [comment]
I am open for any suggestions regarding the names

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -319,11 +319,12 @@
 	departments = SIMPLEDEPT(DEPARTMENT_MEDICAL)
 	department_flag = MEDSCI
 	faction = "Station"
+	alt_titles = list("Medic Trainee", "Surgical Resident")
```
Comment (SleepyGemmy) [inline code/game/jobs/job/medical.dm:322]:
```suggestion
	alt_titles = list("First Responder Intern", "Surgeon Intern")
```

File: code/game/jobs/job/medical.dm [hunk 2]
```diff
@@ -319,11 +319,12 @@
...
 	total_positions = 3
 	spawn_positions = 3
 	supervisors = "the Chief Medical Officer"
 	selection_color = "#15903a"
-	access = list(access_medical, access_surgery, access_medical_equip)
+	access = list(access_medical, access_surgery, access_medical_equip, access_first_responder)
```
Comment (SleepyGemmy) [inline code/game/jobs/job/medical.dm:327]: i'd rather not see interns get FR access. they should be let in, if necessary.

Comment (KingOfThePing) [reply code/game/jobs/job/medical.dm:327]: While I see where this is coming from, why do they get surgery access then?

Comment (SleepyGemmy) [reply code/game/jobs/job/medical.dm:327]: if you want to remove surgery access, feel free.

Comment (KingOfThePing) [reply code/game/jobs/job/medical.dm:327]: Sir yes sir

**Review by SleepyGemmy — APPROVED** [review]
code-wise fine.

**Comment by SleepyGemmy** [comment]
adding do not merge until its known if the PR is wanted or not.

**Comment by JohnWildkins** [comment]
i like alt titles

**Review by JohnWildkins — APPROVED** [review]

**Comment by KingOfThePing** [comment]
The alt-titles are now:

alt_titles = list("First Responder Intern", "Surgeon Intern")

**Pull request merged**